Index: pandastest.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>import requests\n\ndownload_url = \"https://raw.githubusercontent.com/fivethirtyeight/data/master/nba-elo/nbaallelo.csv\"\ntarget_csv_path = \"nba_all_elo.csv\"\n#ArshJain123\nresponse = requests.get(download_url)\nresponse.raise_for_status()    # Check that the request was successful\nwith open(target_csv_path, \"wb\") as f:\n    f.write(response.content)\nprint(\"Download ready.\")\n\nimport pandas as pd\nnba = pd.read_csv(\"nba_all_elo.csv\")\nprint(type(nba))
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/pandastest.py b/pandastest.py
--- a/pandastest.py	(revision dbb597d886748c8bb4ddfe1d246ade11535dcd08)
+++ b/pandastest.py	(date 1674876848840)
@@ -2,7 +2,7 @@
 
 download_url = "https://raw.githubusercontent.com/fivethirtyeight/data/master/nba-elo/nbaallelo.csv"
 target_csv_path = "nba_all_elo.csv"
-#ArshJain123
+
 response = requests.get(download_url)
 response.raise_for_status()    # Check that the request was successful
 with open(target_csv_path, "wb") as f:
